### PR TITLE
Show word count

### PR DESCRIPTION
**Pull request by frostyshadows**
- Show correct word count
- Use milliseconds instead of date for timestamps to make things easier

### CODE REVIEW

File: src/components/WordCountForm.tsx [hunk 1]
```diff
@@ -17,7 +17,8 @@ export default function WordCountForm({ projectTitle }: Props) {
 
   function handleSubmit() {
     alert("A word count was submitted: " + count);
```
Comment (rwblickhan) [inline src/components/WordCountForm.tsx]: Should we get rid of the alert now?

File: src/pages/ExtensionDashboardPage.tsx [hunk 2]
```diff
@@ -7,22 +7,32 @@ import { CREATE_PROJECT_ROUTE, SET_GOAL_ROUTE } from "../App";
...
+    period_start: Date.now(),
   });
+  const [wordCount, setWordCount] = useState<number | null>(0);
   const [projectTitle, setProjectTitle] = useState(PROJECT_TITLE_NONE);
 
   useEffect(() => {
-    loadUserGoal();
-  }, []);
+    loadDashboard();
+    chrome.storage.onChanged.addListener(() => {
```
Comment (rwblickhan) [inline src/pages/ExtensionDashboardPage.tsx:23]: Should this be in a separate effect? We probably don't want it to add another listener every time word count changes

File: src/components/WordCountProgressBar.tsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ export default function WordCountProgressBar({ current, target, period }: Props)
     <div>
       <Progress value={(current / target) * 100} variant="filled" />
       <p>
-        {current} / {target} words {period}
+        {current} / {target} words {period.replace(/["]+/g, "")}
```
Comment (rwblickhan) [inline src/components/WordCountProgressBar.tsx:14]: Why is the `replace` necessary?

**Review by rwblickhan — APPROVED** [review]

File: src/components/WordCountForm.tsx (hunk 1 above)
Comment (frostyshadows) [reply src/components/WordCountForm.tsx]: sure, I'll change it to a console.log

File: src/pages/ExtensionDashboardPage.tsx (hunk 2 above)
Comment (frostyshadows) [reply src/pages/ExtensionDashboardPage.tsx:23]: good catch, this `useEffect` only needs to fire once so I'll get rid of the `wordCount` in the array

File: src/components/WordCountProgressBar.tsx (hunk 3 above)
Comment (frostyshadows) [reply src/components/WordCountProgressBar.tsx:14]: The copy was showing up as "100/500 words "daily""

**Pull request merged**